applications/tari_app_grpc/proto/base_node.proto

```diff
@@ -235,9 +235,8 @@ message MetaData {
     uint64 height_of_longest_chain = 1;
     // The block hash of the current tip of the longest valid chain, or `None` for an empty chain
     bytes best_block = 2;
-    // The number of blocks back from the tip that this database tracks. A value of 0 indicates that all blocks are
-    // tracked (i.e. the database is in full archival mode).
-    uint64 pruning_horizon = 4;
+    // This is the min height this node can provide complete blocks for. A 0 here means this node is archival and can provide complete blocks for every height.
+    uint64 effective_pruned_height = 6;
```

Comment:
This was renamed to `pruned_height`, so better to keep it consistent.

Reply:
Do you want to rename the RPC call to just pruned_height as well?

Reply:
I prefer `pruned_height` as well. Fun fact, I suggested the name `effective_pruned_height` to Yuko a long time ago and I always regretted the long name